src/Tribe/Aggregator.php

```diff
@@ -58,8 +58,7 @@ class Tribe__Events__Aggregator {
 	 */
 	public static function instance() {
 		if ( ! isset( self::$instance ) ) {
```

Comment:
I usually just declare the `private static $instance = false` and check ` if ( false === self::$instance )`

Has the advantage of one less function call. :+1: